src/main/resources/view/WidgetViewBox.fxml

```diff
@@ -1,14 +1,48 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import java.lang.*?>
-<?import java.util.*?>
-<?import javafx.scene.*?>
-<?import javafx.scene.control.*?>
-<?import javafx.scene.layout.*?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
 
-<AnchorPane xmlns="http://javafx.com/javafx"
-            xmlns:fx="http://javafx.com/fxml"
-            fx:controller=""
-            prefHeight="400.0" prefWidth="600.0">
-
-</AnchorPane>
+<StackPane prefHeight="365.0" prefWidth="337.0" styleClass="widget-pane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+   <VBox fx:id="viewBox" prefHeight="301.0" prefWidth="337.0">
+      <children>
+               <StackPane prefHeight="79.0" prefWidth="337.0">
+            <children>
+               <Label fx:id="header" alignment="CENTER" prefHeight="46.0" prefWidth="249.0" styleClass="widget_header"/>
+            </children></StackPane>
+               <StackPane prefHeight="50.0" prefWidth="337.0">
+            <children>
+               <Label fx:id="divOne" prefHeight="17.0" prefWidth="285.0" text="Label" styleClass="widget_div" wrapText="true"/>
+            </children></StackPane>
```

Comment:
I think the indentation and line breaks in this file are off. Can consider taking out the \<children\> tag and leaving it as 
```
<StackPane>
    <Label...>
<StackPane>
```
so that clutter is reduced.